explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -82,8 +81,6 @@ case class SiderealTargetEditor(
 object SiderealTargetEditor:
   private type Props = SiderealTargetEditor
 
-  private given Reusability[Instant] = siderealTargetReusability
```

Comment:
I'm not sure why we were using this reusability logic here. It only refreshed if obs time changed a month or more, which prevented the visualization to update for smaller changes.

Reply:
We didn't want to recalculate AGS unless `obsTime` changed more than a month. I can see why it  could be a problem for visualization. Can we be smarter and make it only use this criteria if not using average PA?

Reply:
Good idea, I'll attempt that. I'll also try to make the logic a bit more explicit, it took me a while to realize we were using a non-standard implicit `Reusability`.